src/main/java/uk/gov/companieshouse/confirmationstatementapi/service/ConfirmationStatementService.java

```diff
@@ -216,7 +216,8 @@ private boolean isConfirmed(SectionDataJson sectionData) {
             var json = confirmationStatementJsonDaoMapper.daoToJson(submission.get());
             return Optional.of(json);
         } else {
-            return Optional.empty();
```

Comment:
I would leave this as it was, Optional.empty() as we shouldnt really use exceptions to control the flow of an app if at all possible.
An empty optional is a submission not found and its up to the calling method to handle that scenario.
Plus by throwing an exception here there is no point us returning an optional as it becomes the only possible return is a full optional.